sidechain/consensus/aura/src/block_importer.rs

```diff
@@ -182,34 +183,38 @@ impl<
 
 	fn get_calls_in_block(
 		&self,
-		sidechain_block: &<SignedSidechainBlock as SignedBlock>::Block,
+		sidechain_block: &SignedSidechainBlock::Block,
 	) -> Result<Vec<TrustedCallSigned>, ConsensusError> {
-		let shard = &sidechain_block.shard_id();
-		let top_hashes = sidechain_block.signed_top_hashes();
-		let calls = self
+		let shard = &sidechain_block.header().shard_id();
+		let top_hashes = sidechain_block.block_data().signed_top_hashes();
+		let tops = self
 			.top_pool_executor
 			.get_trusted_calls(shard)
 			.map_err(|e| ConsensusError::Other(format!("{:?}", e).into()))?;
 
+		let calls = tops
+			.iter()
+			.filter_map(|top| top.to_call())
+			.map(|top| top.to_owned())
+			.collect::<Vec<_>>();
```

Comment:
Here's a change introduced to convert `TrustedOperation` -> `TrustedCallSigned` since the `fn get_trusted_calls` return type change as part of https://github.com/integritee-network/worker/commit/00231051684c4563038b9e5c66077b38db79353e